colab-chat/colab-server
# Inital flask skeleton

This PR adds a bare flask hello world with the standard directory structure.

### To Test

* Code review
* Run locally and confirm that the web page is displayed as expected.
